tools/gn

```diff
@@ -646,10 +646,9 @@ def to_gn_args(args):
   if args.unoptimized and args.target_os == 'android' and args.android_cpu == 'arm64':
     enable_vulkan_validation = True
 
-  if enable_vulkan_validation or (args.enable_impeller_vulkan and
-                                  runtime_mode == 'debug' and
-                                  gn_args['target_cpu'] == 'arm64'):
+  if enable_vulkan_validation:
     gn_args['enable_vulkan_validation_layers'] = True
+    gn_args['impeller_enable_vulkan_validation_layers'] = True
```

Comment:
Why do we have both `enable_vulkan_validation_layers` and `impeller_enable_vulkan_validation_layers`?

Or rather, what's the difference between `impeller_enable_vulkan_validation_layers` and `(enable_impeller_vulkan && enable_vulkan_validation_layers)`

(Not sure that `enable_impeller_vulkan` is the right spelling, but there's probably something we already have that could go there?)

Reply:
`enable_vulkan_validation_layers` is for the fuchsia code, `impeller_enable_vulkan_validation_layers` is for impeller.  There is also a `angle_enable_vulkan_validation_layers`.  `enable_vulkan_validation_layers` predates impeller.  I tried to share the variables.  That's how it was previously, but we need a different default value.  That's why we need a different variable here.